core/startos/src/net/net_controller.rs

```diff
@@ -209,6 +209,21 @@ impl NetService {
             .await?;
         self.update(id, host).await
     }
+    pub async fn clear_bindings(&mut self) -> Result<(), Error> {
+        let package_id = &self.id;
+        self.net_controller()?
+            .db
+            .mutate(|db| {
+                db.as_public_mut()
+                    .as_package_data_mut()
+                    .as_idx_mut(package_id)
+                    .or_not_found(package_id)?
+                    .as_hosts_mut()
+                    .ser(&Default::default())
```

Comment:
this is probably too aggressive, we probably want to keep addresses. just wipe out the binds on each host

Reply:
also, `update` should be called after to do the actual network changes

Reply:
Logic is still not right here, but pretty sure at the moment we don't use this